solr/core/src/test/org/apache/solr/cloud/OverseerModifyCollectionTest.java

```diff
@@ -42,29 +39,27 @@ public static void setupCluster() throws Exception {
   }
 
   @Test
+  @SuppressWarnings({"unchecked"})
```

Comment:
We try to avoid writing code in a way that has warnings so that we needn't suppress them.  Sometimes it's not avoidable.  I can tell here that it's because you called `SolrTestCaseJ4.map`.  Let's call Map.of instead.  Or even Collections.singletonMap because we might take this change to 8x and Java 8 doesn't have Map.of.